packages/node-cli/package-lock.json

```diff
@@ -3132,9 +3131,20 @@
 			}
 		},
 		"vm2": {
-			"version": "3.9.5",
-			"resolved": "https://registry.npmjs.org/vm2/-/vm2-3.9.5.tgz",
-			"integrity": "sha512-LuCAHZN75H9tdrAiLFf030oW7nJV5xwNMuk1ymOZwopmuK3d2H4L1Kv4+GFHgarKiLfXXLFU+7LDABHnwOkWng=="
+			"version": "3.9.8",
+			"resolved": "https://repository.eng.netsuite.com/content/repositories/npmjs/vm2/-/vm2-3.9.8.tgz",
```

Comment:
Please do the `npm install` disconnected from the vpn to update the package-lock.json without reference to the our internal repository.